docs/data/base/components/slider/slider.md

```diff
@@ -0,0 +1,60 @@
+---
+product: base
+title: React Slider unstyled component and hook
+components: SliderUnstyled
+githubLabel: 'component: slider'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#slider
+packageName: '@mui/base'
+---
+
+# Slider
+
+<p class="description">Sliders allow users to make selections from a range of values.</p>
+
+Sliders show a range of values along a bar, from which users may select a single value.
+They are ideal for adjusting settings such as volume, brightness, or applying image filters.
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+```js
+import SliderUnstyled from '@mui/base/SliderUnstyled';
+```
+
+{{"demo": "UnstyledSlider.js", "defaultCodeOpen": false}}
+
+## Discrete sliders
+
+Discrete sliders snap to the nearest step.
+You can generate a mark for each step with `marks={true}`.
+
+{{"demo": "DiscreteSlider.js"}}
```

Comment:
```suggestion
{{"demo": "DiscreteSlider.js", "defaultCodeOpen": false}}
```

Reply:
In this case (and below), we're showing the usage of a prop, not a method of customization. The code that is described is included in the preview, so I decided to leave the preview visible.